gui/wxpython/datacatalog/tree.py

```diff
@@ -966,6 +999,21 @@ def OnDeleteLocation(self, event):
                    showTraceback=False)
         self.ReloadTreeItems()
 
+    def OnDownloadLocation(self, event):
+        """
+        Download location online
+        """
+        try:
+            location = download_location_interactively(
+                    self, self.selected_grassdb[0].data['name']
+                    )
+            if location:
+                self.ReloadTreeItems()
+        except Exception as e:
```

Comment:
Catching such a generic exception (Exception) seems unnecessary. The error message for many exceptions will not be helpful for the end user. Was the original code doing that? What do you expect from `download_location_interactively()`? Are some issues handled there already? For a fallback mechanism for unexpected errors, the default mechanism of reporting them in the console is probably enough.

Reply:
Generic exceptions removed. Not needed here.